README.md

```diff
@@ -171,7 +171,11 @@ Import the certificate to Firefox:
 
 This project is configured for multiple forms of tests: unit, integration, and e2e. Compilation and linting occurs as a pre-commit hook, and all tests are run as a pre-push hook.
 
-Some tests require an OpenAM instance with a public OAuth client configured. Specify your environment details in an `.env` file:
+### E2E Test Requirements
+
+Node v13.10 or higher is required to run the mock E2E server application.
```

Comment:
To simplify writing the mock server(s), I wanted to leverage ES Modules, like TypeScript but without the compile step, and that required Node v13.10. Now that everything is built, I could migrate to TypeScript, but I'm not sure how much value it provides other than language consistency.